filament/src/materials/blur.mat

```diff
@@ -0,0 +1,108 @@
+material {
+    name : blur,
+    parameters : [
+        {
+            type : sampler2d,
+            name : ssao,
+            precision: medium
+        },
+        {
+            type : sampler2d,
+            name : depth,
+            precision: medium
+        },
+        {
+            type : float4,
+            name : resolution
+        },
+        {
+            type : int2,
+            name : axis
+        }
+    ],
+    variables : [
+        vertex
+    ],
+    vertexDomain : device,
+    depthWrite : false,
+    depthCulling : true,
+    shadingModel : unlit,
+    variantFilter : [ skinning, shadowReceiver ],
+    culling: none
+}
+
+vertex {
+    void materialVertex(inout MaterialVertexInputs material) {
+        // far-plane in view space
+        vec4 position = getPosition(); // clip-space
+        position.z = 1.0; // far plane
```

Comment:
Remove, it's not used here

Reply:
Well, I want to make sure that z=1.0 -- I guess it's guaranteed by the data provided on the cpu side, but I wanted to make it explicit in the shader. z=1.0 is needed for depth culling the skybox.

Reply:
But you're modifying by value, it's not going to do anything.

Reply:
uh. lol. yeah.